### ISSUE TITLE
Importing a fridge tag fails if an import has happened in another store

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#4755]
## What went wrong? 😲

Given:
* you are running postgres
* there are mutliple stores configured for your site
* a fridge tag has been imported in another store

Then:
any fridge tag import will fail

The issue is that this call to check matching sensor config (berlinger.rs, line 204)

```
let result = get_matching_sensor_breach_config(
        connection,
        &sensor_row.store_id,
        temperature_breach_config,
        &breach_row_type,
    )?;

    if !result.is_empty() {
        return Ok(());
    };
```

is checking for a config based on store_id, duration_milliseconds, minimum_temperature, maximum_temperature and type.
However, the constraint on the table is for a unique description and this does not include the store_id, so the insert following the code above will fail.

## Expected behaviour 🤔
Able to import the fridge tag data for a new store.

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to Cold chain / monitoring
2. Import a fridge tag
3. Switch to another store and repeat

## Your environment 🌱

- Open mSupply Version:
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

**Comment by lache-melvin** [comment]
Setting as normal as I don't believe there are any deployed multi-store sites, but you should be able to do this!

**Comment by lache-melvin** [comment]
See PR for needs refinement context 🙏

**Comment by jmbrunskill** [comment]
Refinement:

If you import a fridge tag and you're logged into a store, other than the one associated with the sensor.
Duplicated configs for different stores is ok...

**Comment by adamdewey** [comment]
> 
Yes, you shouldn't be able to import the same fridgetag into a different store (you'd need to move it to the new store first), but you should definitely be able to import a different fridgetag

How do you move a Fridge-tag to a different store?

**Comment by adamdewey** [comment]
> Hmm, I was thinking of the "Set location" button when viewing a sensor on desktop mSupply, but just realised that locations are specific to a store, and you can only see sensors associated with the store you're logged into => you can only ever move it to another location in the same store!

Ah, I wonder if this is also what happened for me on the oms demo server. 

I had sensors (not Fridge-tag) associated with a store, then switched them to a new store on the same site. 

The `Chart` pages shows the data ok, but the sensors don't appear in the `Sensors` page

![image](https://github.com/user-attachments/assets/d8e658d2-3614-4463-8452-ea3967624735)

![image](https://github.com/user-attachments/assets/cf282e85-f11b-477f-8b62-e5ce0349f4f4)

**Comment by alainsussol** [comment]
@jmbrunskill 
Hi James,

Was the fix for this included in 2.4.1 ?

**Comment by jmbrunskill** [comment]
@alainsussol my change is in 2.5.0, I'm not sure it's worth doing much testing with, maybe just test that you can still import?
We probably need to do some other business logic changes such a allowing the sensor to move stores, or telling the user they can't import into the wrong store perhaps? I haven't created a follow up issue as there's no focus or budget on this so we don't want to spend much time on it...
